src/main/java/io/redisearch/Schema.java

```diff
@@ -21,25 +21,40 @@
     }
 
     public static class Field {
+        private final FieldName fieldName;
         public final String name;
```

Comment:
@sazzad16 do we still need this field?
```
        public final String name;
```

Reply:
@gkorland No. But removing this would break the backward compatibility.